examples/cifar10_alex.py

```diff
@@ -0,0 +1,117 @@
+'''Train a simple deep CNN on the CIFAR10 small images dataset.
+In this example, we use LocallyConnected2D to reproduce Alex's implement
+There are two difference, one is input size (32x32 vs. 24x24), the other 
+is replacing LRN with BatchNormalization.
+
+GPU run command:
+    THEANO_FLAGS=mode=FAST_RUN,device=gpu,floatX=float32 python cifar10_cnn.py
+
+It gets down to 0.65 test logloss in 25 epochs, and down to 0.55 after 50 epochs.
```

Comment:
What is the final accuracy of the model; are you able to reproduce the results of the original model?